ci-operator/step-registry/vsphere-acm-virt/cold-migration/vsphere-acm-virt-cold-migration-commands.sh

```
@@ -0,0 +1,375 @@
+
+if [[ "${succeeded}" != 'True' ]]; then
+    : "Migration timed out after ${MIGRATION_TIMEOUT}s"
+    DumpDiagnostics
+    exit 1
+fi
+
+# --------------------------------------------------------------------------
+# Validate migrated VM is Running on OCP
+# --------------------------------------------------------------------------
+typeset vmiPhase=''
+typeset -i vmiDeadline=$(( SECONDS + 300 ))
```

Shellcheck (0.11.0)

[info] SECONDS was modified in a subshell. That change might be lost.

(SC2031)